robolectric-utils/src/main/java/org/robolectric/util/AccessibilityUtil.java

```diff
@@ -0,0 +1,129 @@
+package org.robolectric.util;
+
+import com.google.android.apps.common.testing.accessibility.framework.AccessibilityCheck;
+import com.google.android.apps.common.testing.accessibility.framework.AccessibilityCheckPreset;
+import com.google.android.apps.common.testing.accessibility.framework.AccessibilityCheckResultUtils;
+import com.google.android.apps.common.testing.accessibility.framework.AccessibilityViewCheck;
+import com.google.android.apps.common.testing.accessibility.framework.AccessibilityViewCheckResult;
+import com.google.android.apps.common.testing.accessibility.framework.AccessibilityViewHierarchyCheck;
+import com.google.android.apps.common.testing.accessibility.framework.AccessibilityCheckResult.AccessibilityCheckResultType;
+
+import android.view.View;
+
+import org.robolectric.annotation.AccessibilityChecks;
+
+import java.io.PrintStream;
+import java.lang.reflect.Method;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Utility class for checking Views for accessibility
+ */
+public class AccessibilityUtil {
+  private AccessibilityUtil() {}
+
+  /**
+   * Check a {@code View} for accessibility. Only performs checks if 
+   * accessibility is enabled using an {@link AccessibilityChecks} annotation
+   * or the environment variable {@code a11y_checks_enabled} is set to 
+   * {@code true}. {@link AccessibilityChecks} values override those from 
+   * environment variables.
+   * 
+   * @param view The {@code View} to examine
+   * @return {@code false} if accessibility checks are enabled and a problem
+   * was found. {@code true} otherwise.
+   */
+  public static boolean passesAccessibilityChecksIfEnabled(View view) {
+    boolean checksEnabled = false;
+
+    /* Pull defaults from environment variables */
+    String checksEnabledString = System.getenv("a11y_checks_enabled");
```

Comment:
This should be named something like "robolectric.accessibility.check" to match the other system properties that Robolectric uses. See http://robolectric.org/configuring/ for other configuration options.